src/styles/_theme.scss

```diff
@@ -1,6 +1,6 @@
+@use '~@angular/material' as mat;
 // Copyright 2021 Carnegie Mellon University. All Rights Reserved.
 // Released under a MIT (SEI)-style license. See LICENSE.md in the project root for license information.
-@import '~@angular/material/theming';
```

Comment:
Why remove this import? Seems like a step back to specify the rgba for each color individually instead of using the variables. Adding this import back lets that stay the same. I'm guessing the ng update for Material did this? Doesn't feel right though.